src/styles/index.scss

```diff
@@ -1,3 +1,77 @@
+$hand-transform-origin: center bottom;
+
+@mixin center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@mixin starting-position {
```

Comment:
It would be better to name your mixin more descriptively. The name `starting-position` does not clearly convey its purpose. Consider renaming it to something like `absolute-center-bottom`.